zipline/pipeline/pipeline.py

```diff
@@ -54,7 +60,7 @@ def screen(self):
         """
         return self._screen
 
-    @expect_types(term=Term, name=str)
+    @expect_types(term=ComputableTerm, name=str)
```

Comment:
I would try to re-use the error message above rather than using `expect_types` here.  This will spit out a message telling the user that we expected a `ComputableTerm` but got a `BoundColumn`, which won't mean anything to most users.